frontend/app/dashboard/src/views/members/MembersMenu.vue

```diff
@@ -276,16 +277,38 @@ async function upgradePeriod() {
     await openPeriod(platform.value.period);
 }
 
+async function getPeriods() {
+    try {
+        await organizationManager.value.loadPeriods(false, false, owner);
+    }
+    catch (e) {
+        console.error(e);
+        return [];
+    }
+
+    const periods = organizationManager.value.organization.periods?.organizationPeriods;
+    if (periods === undefined) {
+        return [organizationManager.value.organization.period];
+    }
+
+    const periodsCopy = [...periods];
+
+    periodsCopy.sort((a, b) => Sorter.byDateValue(a.period.startDate, b.period.startDate));
+
+    return periodsCopy;
+}
+
 async function editMe() {
     if (!$rootCategory.value) return;
+
     await present(new ComponentWithProperties(NavigationController, {
         root: new ComponentWithProperties(EditCategoryGroupsView, {
             category: $rootCategory.value,
-            period: period.value,
+            periodId: period.value.id,
+            periods: await getPeriods(),
```

Comment:
Een async operatie, maar je doet niets om progress te tonen. Als er geen internet is, lijkt het alsof de knop niet gaat reageren. Voor await present hoef je dat niet te doen, omdat dat altijd gegarandeerd is, maar ovor netwerk requests moet er altijd een loading indication zijn.

In dit geval moet je daarvoor een promise view gebruiken via bv. `PromiseComponent`